packages/third-parties/bullmq/src/contracts/JobMethods.ts

```diff
@@ -1,3 +1,5 @@
-export interface JobMethods {
-  handle(payload: unknown): unknown;
+import {Job} from "bullmq";
+
+export interface JobMethods<DataType = any, ReturnType = unknown> {
```

Comment:
```suggestion
export interface JobMethods<DataType = unknown, ReturnType = unknown> {
```

I think we should use `unknown` here too.